src/ncp/spinel_decoder.cpp

```diff
@@ -35,6 +35,7 @@
 #include "spinel_decoder.hpp"
 
 #include "common/code_utils.hpp"
+#include "utils/wrap_string.h"
```

Comment:
It's ok to add this header file but technically it is not be needed.

- `spinel_encoder.hpp` is included here which in turn includes `spinel.h`.
- `spinel.h` expects all platform related header files  to be defined in `SPINEL_PLATFORM_HEADER` which should include `<stdarg.h>`, `<stdbool.h>`, `<stdint.h>`, ... , `<string.h>` or their equivalent on the platform.
- In OpenThread `SPINEL_PLATFORM_HEADER` is mapped to [`spinel_platform.h`](https://github.com/openthread/openthread/blob/master/src/ncp/spinel_platform.h#L31) which `#include "utils/wrap_string.h"`